### ISSUE TITLE
Angular Console needs port 7777

### ISSUE THREAD

**Issue by patrickhousley** [nrwl/angular-console#155]
When loading on Mac, the screen is blank. See attached screenshot.

![screen shot 2018-08-14 at 12 45 10 pm](https://user-images.githubusercontent.com/3984979/44108472-25648a2e-9fc0-11e8-9714-512d2b17d7bb.png)

Mac OS 10.12.6

```
phousley@phousley-mac ~/projects/
$ nvm ls
->      v8.11.2
default -> 8 (-> v8.11.2)
node -> stable (-> v8.11.2) (default)
stable -> 8.11 (-> v8.11.2) (default)
iojs -> N/A (default)
lts/* -> lts/carbon (-> v8.11.2)
lts/argon -> v4.9.1 (-> N/A)
lts/boron -> v6.14.2 (-> N/A)
lts/carbon -> v8.11.2
```

**Comment by vsavkin** [comment]
@patrickhousley could you run the app in the terminal, so we can see the logs written by Electron?

**Comment by patrickhousley** [comment]
```
phousley@phousley-mac ~/projects
 [ master | ✔ ] $ /Applications/Angular\ Console.app/Contents/MacOS/Angular\ Console
starting server on port 7777
Checking for update
Update for version 6.0.0-beta.1 is not available (latest version: 6.0.0-beta.1, downgrade is disallowed).
checkForUpdates is called. downloadPromise is null.
```

**Comment by patrickhousley** [comment]
@vsavkin Is there anything else you need to unblock this?

**Comment by mike-park** [comment]
ditto for me, blank screen on Mac, no tasks to run on a newly created project. Started from terminal:

```
pro13:ng-console mikep$ /Applications/Angular\ Console.app/Contents/MacOS/Angular\ Console
starting server on port 7777
Checking for update
Update for version 6.0.0-beta.2 is not available (latest version: 6.0.0-beta.2, downgrade is disallowed).
checkForUpdates is called. downloadPromise is null.
```

**Comment by vsavkin** [comment]
@patrickhousley when launching Angular Console we start a node server that supplies the app with the needed data. It looks like the server isn't up for you. Can you check the network tab in the dev tools to confirm that?

**Comment by patrickhousley** [comment]
<img width="557" alt="screen shot 2018-09-28 at 9 55 40 am" src="https://user-images.githubusercontent.com/3984979/46216224-b6441200-c304-11e8-9877-72af2f1f70c5.png">

**Comment by patrickhousley** [comment]
```
phousley@phousley-mac ~
$ lsof -nP -i4TCP:7777 | grep LISTEN
com.docke  1391 phousley   20u  IPv4 0xe0aa4be8aa2766ef      0t0  TCP *:7777 (LISTEN)
Angular   97506 phousley   39u  IPv6 0xe0aa4be8bf89f4bf      0t0  TCP *:7777 (LISTEN)
```

I feel a little stupid now, lol. So I have a docker machine for work that has to use port 7777. Is there any way the app could use a random port? Or maybe check if 7777 is in use and have a fallback.

**Comment by mrmeku** [comment]
@patrickhousley Picking a random port should not be a problem at all! We will definitely do so as a fallback. Its nice to have a non-random default since it allows us to debug for people easier, but in your case it shouldn't just block you from opening the app

**Comment by mrmeku** [comment]
This is done now!
